### DISCUSSION TITLE
[Import as Image] Adding of "Original Image" instead of applying as Featured Image

### DISCUSSION THREAD

**Discussion by mavproductions** [mealie-recipes/mealie#4291] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

When using OpenAI upload as image, the featured image is the original image used during the upload.
![image1](https://github.com/user-attachments/assets/95f74e19-4826-4f23-a53c-21635fa5792b)

Ideally the featured image is something to showcase the final product of the recipe, whether scraped from the original image, or manually uploaded by the uploader.

When inspecting the image, it also appears impossible (for me😅) to grab a direct image link to verify the OpenAI data scrape was 100% accurate. It just shows "garbly-goo".
![image3](https://github.com/user-attachments/assets/8e36420c-e301-4c79-86af-075d3ed8570e)

Ideally having a button at the bottom (like "Upload by URL" uploads) saying "Original Image) would not only fix this issue, but free up the featured image by default. 
![image2](https://github.com/user-attachments/assets/4dfe99f2-a31e-4c9f-9505-a1cfc250eca8)

### Please provide a concise description of the feature that would resolve your issue.

Add a bottom button labelled "**Original Image**"

### Please consider and list out some caveats or tradeoffs made in your design decision

Nothing really unless some people like the original image as the featured image. Maybe add a checkbox to assign it as featured image, or a secondary crop page for cropping what part of the original image is to be the featured image?

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by derwassi** [comment]
I'm importing quite a few recipes from a few Magazins and the Post Processing ist quite tedious. 
To elaborate on yout Initial Idea, i think it would be good to automatically add the original image as Attachment (of Type recipe, for Future reference in Case of parsing errors) and to be able to select two crop-sections of the original image: one with the Text, for openai, one for the Image to show in the recipe. 
Furthermore it would be great to specify Tags and Type of the imported recipe and jump directly to ingredient parsing after Upload. This would Streamline the process of importing a recipe incredibly.
Specifying Tags/Type beforehand and switchung directly to ingredient Parser would be benefitial for all import types imho.

If this is a Feature that i probably could Provider a pull requests for, If it gets accepted
